qiita_db/user.py

```diff
@@ -463,6 +468,45 @@ def _change_pass(self, newpass):
         conn_handler = SQLConnectionHandler()
         conn_handler.execute(sql, (hash_password(newpass), self._id))
 
+    def add_message(self, message):
+        """Add a message to the user's message queue
+
+        Parameters
+        ----------
+        message : str
+            Message to add to queue
+        """
+        count = r_client.lpush("%s:messages" % self._id, '%s\n%s' %
+                               (message, datetime.now()))
+        if count > 100:
+            # only store 100 messages, so pop oldest
+            r_client.rpop()
+
+    def messages(self, count=100, as_html=False):
```

Comment:
I disagree on having any kind of HTML in qiita_db. Please, remove this. It is easy enough on the interface to do the replaces that you are doing here. (Note that this is already done in other places in the code)